tests/test_common_features.py

```diff
@@ -1,12 +1,27 @@
+import functools
 from pytest_dash.utils import (
     import_app,
     wait_for_element_by_id,
 )
 
 
 def access_demo_app(dash_threaded, selenium, app_name):
-    """mimic a user click on the app link from the gallery"""
+    """Mimic a user click on the app link from the gallery."""
     dash_bio_index = import_app('index')
     dash_threaded(dash_bio_index)
```

Comment:
Might I suggest using `dash_subprocess` instead, it's faster and more stable if you don't need to have the app instance. Lock `pytest-dash>=1.1.0` since it has a fix for using it on circle.

Reply:
As discussed on private slack, experimentation showed that dash_threaded was faster in that case.